crates/cairo-addons/src/vm/runner.rs

```diff
@@ -208,9 +220,21 @@ except Exception as e:
         Ok(PyDictManager { inner: dict_manager })
     }
 
-    fn run_until_pc(&mut self, address: PyRelocatable, resources: PyRunResources) -> PyResult<()> {
-        let mut hint_processor =
-            HintProcessor::default().with_run_resources(resources.inner).build();
+    #[pyo3(signature = (address, resources, enable_pythonic_hints=false))]
+    fn run_until_pc(
```

Comment:
if the option is given at `new` I don't think that we should have a second time the param. What happens if you given False in new and True here?

Reply:
added as a field in the runner, set upon new